components/molecules/BreadcrumbMenu.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+class BreadcrumbMenu extends Component {
+    static displayName = 'BreadcrumbMenu';
+
+    static propTypes = {
+        path: PropTypes.string.isRequired,
+        menu: PropTypes.arrayOf(
+            PropTypes.shape({
+                title: PropTypes.string.isRequired,
+                path: PropTypes.string.isRequired
+            })
+        ).isRequired,
+        LinkElt: PropTypes.oneOfType([PropTypes.element, PropTypes.func]).isRequired
+    };
+
+    state = {
+        showMenu: false
+    };
+
+    _toggleMenu = () => this.setState(({ showMenu }) => ({ showMenu: !showMenu }));
+
+    render() {
+        const { showMenu } = this.state;
+        const { path: parentPath, LinkElt, menu } = this.props;
+        const menuClass = cx('gds-bubble__menu', 'gds-bubble__menu--left', 'gds-bubble__menu--sm', {
+            'gds-bubble__menu--menu-open': showMenu
+        });
+
+        return (
+            <div onClick={this._toggleMenu} className="gds-page-header__breadcrumbs-menu">
+                <div className="gds-page-header__breadcrumbs-menu-dots" />
+                <div className={menuClass}>
+                    <ul className="gds-bubble__menu-list">
+                        {menu.map(({ title, path }) => (
+                            <li key={path} className="gds-bubble__menu-list-item -ellipsis">
+                                <LinkElt
+                                    className="gds-bubble__menu-list-link"
+                                    to={`${parentPath}/${path}`}>
+                                    {title}
+                                </LinkElt>
+                            </li>
+                        ))}
+                    </ul>
+                </div>{' '}
```

Comment:
is this `{' '}` dingleberry intentional? if so, maybe we should wrap it in a `<span>` so it at least feels intentional.

Reply:
it is intentional, It was added by prettier... maybe? Good idea, the span works just fine. I'll push it.

Reply:
done